comms/dht/src/crypt.rs

```diff
@@ -55,6 +55,9 @@ use crate::{
 pub struct CipherKey(chacha20::Key);
 pub struct AuthenticatedCipherKey(chacha20poly1305::Key);
 
+const LITTLE_ENDIAN_U64_SIZE_REPRESENTATION: usize = 8;
+const MESSAGE_BASE_LENGTH: usize = 128;
```

Comment:
A partial transaction message in my test was 4111 bytes, to obfuscate this from other messages, we'll need to make every message bigger than this. I suggest a padding of 6000.